### DISCUSSION TITLE
同步不同设备间的设置-dotfiles

### DISCUSSION THREAD

**Discussion by caibingcheng** [caibingcheng/comment#303] [Announcements]
https://imcbc.cn/202210/summary-dotfiles/

**Comment by penndu** [comment]
来你这里评论需要佛系，因为不是每次都能加载成功~

**Reply by penndu** [reply]
夜间模式字体颜色有些问题！

**Comment by ryan4yin** [comment]
话说我最近折腾了个 NixOS 系统与其包管理器 Nix，这玩意儿可以当成 dotfiles 的进阶版用，更强大当然也更复杂，有兴趣可以看一看~

**Reply by caibingcheng** [reply]
我有看过你的文章，大概了解了一下nixos，这种想法还是挺有意思的
